web/src/pages/api/post/search.ts

```diff
@@ -83,11 +91,17 @@ export default async function search(
       take: limit ? parseInt(limit.toString()) : undefined,
       skip: cursorId ? 1 : 0,
       cursor: cursorId ? { id: cursorId.toString() } : undefined,
-      orderBy: {
-        likes: {
-          _count: "desc",
-        },
-      },
+      orderBy:
+        recentOnly === "true"
+          ? { dateCreated: "desc" }
+          : [
+              {
+                likes: {
+                  _count: "desc",
+                },
+              },
+              { dateCreated: "desc" },
+            ],
```

Comment:
This is the fix. I essentially especiafied an array of order values and prisma will run a query that sorts it first by likes and then by dateCreated. I thought this was not possible but seems like it just isnt well documented that you can order by more than one field like this

Reply:
Down the line we may want to do some kind of formula that deprioritizes older highly liked posts so it isn't always the most liked at the top, but sounds good for now